utils/resizer/src/index.js

```diff
@@ -1,19 +1,58 @@
+import { RESIZER_TOKEN_VERSION } from 'fusion:environment'
+
+import imageANSToImageSrc from '@wpmedia/arc-themes-components/src/utils/image-ans-to-image-src'
+
+import calculateWidthAndHeight from './calculateWidthAndHeight'
+
+const formatSrc = (srcWithResizerUrl, resizedOptions) => {
+  return srcWithResizerUrl.concat(
+    '?',
+    new URLSearchParams({
+      ...resizedOptions,
+      width: Math.floor(resizedOptions.width),
+      height: Math.floor(resizedOptions.height),
+    }).toString(),
+  )
+}
+
 export function buildResizerURL(
   _url,
   resizerKey,
   resizerURL,
   width = 0,
   height = 0,
+  ansImage,
 ) {
-  if (_url && typeof window === 'undefined') {
-    const Thumbor = require('thumbor-lite')
-    const thumbor = new Thumbor(resizerKey, resizerURL)
-    let imgSrc = _url.replace(/^http[s]?:\/\//, '').replace(' ', '%20')
-    if (imgSrc.includes('?')) {
-      imgSrc = imgSrc.replace('?', '%3F')
-    }
+  if (ansImage?.auth?.[RESIZER_TOKEN_VERSION]) {
```

Comment:
Is this the right conditional?

Like we're checking if the auth token is already on it on the ansImage under auth.RESIZER_TOKEN_VERSION, but if it is then below we end up adding it under auth, but directly as the auth value?